### DISCUSSION TITLE
Download MQTTSubscribe- missing URL error

### DISCUSSION THREAD

**Discussion by GFRANKG** [bellrichm/WeeWX-MQTTSubscribe#183] [General]
I get "wget: missing URL"  when starting the install process..

I used : wget -P $DOWNLOAD_ROOT https://github.com/bellrichm/WeeWX-MQTTSubscribe/archive/vX.Y.Z.tar.gz

I tried updating the link to :https://github.com/bellrichm/WeeWX-MQTTSubscribe/archive/refs/tags/v2.2.2.tar.gz"

No luck...   not entirely sure why this is happening to me??

**Comment by bellrichm** [comment]
The following worked for me
```
wget -P /tmp https://github.com/bellrichm/WeeWX-MQTTSubscribe/archive/refs/tags/v2.2.2.tar.gz
```
Obviously you want to change /tmp to a directory of your choice.

**Comment by GFRANKG** [comment]
Thank you bellrichm,   that worked for me!

**Comment by GFRANKG** [comment]
Thanks for the super fast solution!
